tests/conftest.py

```diff
@@ -22,6 +24,23 @@
     "conda.testing.fixtures",
 )
 
+log = getLogger(__name__)
+
+
+@pytest.fixture(name="monkeypatch")
```

Comment:
Are we simply shadowing the original `monkeypatch` with this? Can we add this to an importable location in `conda.testing`? This will be useful in other projects like `conda-libmamba-solver`.

Reply:
Actually we should move all these fixtures to somewhere in `conda.testing`. I had to copy a few over to conda-libmamba-solver to make the new test style work.

Reply:
Done, see https://github.com/conda/conda/pull/13449/commits/e00239cc839d810a0b1ecfabea62743d82092224